.github/workflows/website-deploy.yml

```diff
@@ -30,6 +25,7 @@ jobs:
           pip install mkdocs-material=="8.*"
           pip install mike
           pip install mkdocs-macros-plugin
+          pip install lunr
```

Comment:
why this is needed?

Reply:
the action's logs contain a warning about missing lunr. In general lunr is a simple search agent.